### PR TITLE
Feature/widgets for engine and workers

### PR DESCRIPTION
**Pull request by shaniacht1**
related to https://github.com/demisto/etc/issues/12053
DO NOT MERGE YET

TODO before merge:
- [ ] add integration count per engine/engine group - won't be done now
- [ ] on client side - timeFrame should change according to dateRange - won't be done now
- [x] server side - save data according to server config
- [x] client side - display only the data available (if 60 minutes returned, display 60 minutes and not more)
- [x] add size as limit for maximum time displayed (?)

### CODE REVIEW

**Comment by dorsha** [comment]
@shaniacht1 what about this one?

**Comment by shaniacht1** [comment]
@dorsha waiting the @kirbles19 to approve the names and descriptions of the widgets.

**Comment by kirbles19** [comment]
Done editing.

File: Widgets/widget-CPUPercentageTrend.json [hunk 1]
```diff
@@ -20,5 +21,5 @@
     "params": {
         "timeFrame": "minutes"
     },
-    "size": 61
-}
\ No newline at end of file
+    "size": 1500
```
Comment (Bars92) [inline Widgets/widget-CPUPercentageTrend.json]: this change is ok for 3.5.1?

File: Widgets/widget-EngineTotalWorkersBar.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+    "id": "engine-total-workers-bar",
+    "version": -1,
+    "fromVersion": "4.0.0",
+    "modified": "2018-03-14T17:42:44.919467+02:00",
+    "name": "Engine Workers Count",
+    "description": "Current number of workers per engine",
```
Comment (Bars92) [inline Widgets/widget-EngineTotalWorkersBar.json]: current should be total no?

File: Widgets/widget-CPUPercentageTrend.json (hunk 1 above)
Comment (shaniacht1) [reply Widgets/widget-CPUPercentageTrend.json]: yep

**Pull request merged**